docs/dal/index.rst

```diff
@@ -146,10 +146,168 @@ Table Access Protocol
 
     -- `Table Access Protocol <https://www.ivoa.net/documents/TAP/>`_
 
+Getting started
```

Comment:
There is already a getting started heading in this very same document, I strongly suggest not to repeat it (it also messes up the heading references)